analyzer/modules/accounts.go

```diff
@@ -29,18 +27,13 @@ func NewAccountsHandler(source storage.RuntimeSourceStorage, qf *analyzer.QueryF
 
 // PrepareAccountsData prepares raw data from the `accounts` module for insertion.
 // into target storage.
-func (h *AccountsHandler) PrepareData(ctx context.Context, round uint64, batch *storage.QueryBatch) error {
```

Comment:
Big :heart: for the loss of `ctx`. I very clearly proves the separation of data fetching from data processing that we implemented a while ago in consensus but kept delaying in runtimes.